thirdparty/README.md

```diff
@@ -990,6 +990,10 @@ Files extracted from upstream source:
 Versions of this SDK do not have to match the `vulkan` section, as this SDK is required
 to generate Metal source from Vulkan SPIR-V.
 
+Patches:
+
+- `0001-fix-argument-buffer-access.patch`
```

Comment:
```suggestion
- `0001-fix-argument-buffer-access.patch` (GH-108028)
```

Reply:
I've removed the patch and updated SPIRV-Cross, as [it has been fixed](https://github.com/KhronosGroup/SPIRV-Cross/pull/2503)